app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```

Comment:
Cool, I've added tweaked the application.rb config file to avoid that in the future with the generate command.